actionview/lib/action_view/helpers/form_tag_helper.rb

```diff
@@ -134,13 +134,15 @@ def select_tag(name, option_tags = nil, options = {})
 
         if options.include?(:include_blank)
           include_blank = options.delete(:include_blank)
+          options_for_blank_options_tag = { value: '' }
 
           if include_blank == true
             include_blank = ''
+            options_for_blank_options_tag[:label] = ' '
```

Comment:
@jeremy its better than specifying an empty value or content of the tag, label is the least path of resistance, so that we don't mix up with actual non-empty whitespace values elsewhere.
For example from http://codepen.io/newtron/pen/PZaxzO, its better than assuming content as &nbsp; than `label=" "` 

http://stackoverflow.com/questions/8605516/default-select-option-as-blank has some opinions about this.